doc/source/docs/user_guide/mapping.rst

```diff
@@ -23,19 +23,21 @@ Loading some example data:
 
 .. ipython:: python
 
-    world = geopandas.read_file(geopandas.datasets.get_path('naturalearth_lowres'))
-    cities = geopandas.read_file(geopandas.datasets.get_path('naturalearth_cities'))
+    import geodatasets
+
+    chicago = geopandas.read_file(geodatasets.get_path("geoda chicago health"))
+    groceries = geopandas.read_file(geodatasets.get_path("geoda groceries"))
 
 You can now plot those GeoDataFrames:
 
 .. ipython:: python
 
-    # Examine country GeoDataFrame
-    world.head()
+    # Examine chicago GeoDataFrame
+    chicago.head()
 
-    # Basic plot, random colors
-    @savefig world_randomcolors.png
-    world.plot();
+    # Basic plot, single color
+    @savefig chicago_singlecolor.png
+    chicago.plot();
```

Comment:
Looking at this in the built docs, I wonder if it makes sense to adjust the figsize (either in code or specifically for the images embedded in the docs - the portrait orientation of the chicago compared to the world map which was landscape just means everything is a little smaller.

Reply:
Made them a bit bigger.